packages/components/src/internal/components/samples/constants.ts

```diff
@@ -202,6 +202,9 @@ export const SAMPLE_STORAGE_COLUMNS_WITH_SUBSELECT_EXPR = [
 
 export const SAMPLE_INSERT_EXTRA_COLUMNS = [...AMOUNT_AND_UNITS_COLUMNS, ...SAMPLE_STORAGE_COLUMNS, ALIQUOTED_FROM_COL];
 
+// those lookup values are at Home project level, no need to reload on target folder change
+export const SAMPLE_ALL_PROJECT_LOOKUP_FIELDS = ['SampleState', 'Units'];
```

Comment:
Reloading with the change in target folder would actually be problematic, right? Does it make sense for these lookups to have their container defined (so this special casing isn't needed)?  Just wondering if we'll have other similar lookups that we forget to add to this list.

Reply:
Reloading is not problematic, just annoying from ux point since it erases grid data. Good idea for checking lookup fields' containerPath. I've made the change to skip reloading if the containerPath for the field is set. We can explore having those bulit-in fields use Home project container lookup path in the future.